docs/source/get_started/installation.rst

```diff
@@ -4,46 +4,33 @@
 Installation
 ============
 
-Installing Enaml is a straight forward process. It requires few external
-dependencies, and those which are required are easily installed, with most
-projects providing binaries or a simple Python setup script. The sections
-below describe how to install Enaml and all of its dependencies from scratch,
-starting with the installation of a Python runtime. The instructions assume
-that the user's system has a C++ compiler and the `Git`_ command line tools
-installed and available on the system path.
-
-.. _Git: http://git-scm.com
+Installing Enaml is a straight-forward process. There are three approaches to
+choose from.
 
+Anaconda: The Easiest Way
+-------------------------
 
-.. topic:: The Easy Way
-
-    If installing and building Enaml and its dependencies from scratch is not
-    appealing, the free (and unaffiliated) `Anaconda`_ Python distribution
-    provides a complete Python environment in which the last release of Enaml
-    can be installed using conda from the conda-forge channel::
+If you use the `Anaconda`_ Python distribution platform, you the latest 
```

Comment:
"You" can be removed in "you the latest"

Reply:
It may also be good to mention that [miniconda](https://conda.io/miniconda.html) can be used as a lightweight alternative to the full anaconda platform.

Reply:
Both fixed in next commit.